lib/python/treadmill_aws/cli/admin/aws/image.py

```diff
@@ -35,13 +35,19 @@ def image():
         '--account', required=False,
         help='Image account, defaults to current.'
     )
+    @click.option(
+        '--tag',
+        multiple=True,
+        required=False,
+        help='Image tag in the format foo,bar'
```

Comment:
it is a bit counter intuitive format for name/value list. we have built-in cli.DICT type (i think it is k=v) so making it multiple will work.

Reply:
Makes sense, updated to use type=cli.DICT instead.